custom/opm/reports.py

```diff
@@ -1208,10 +1210,31 @@ def get_rows(self, datespan):
                 break
         return UsersIdsData(config=config).get_data()
 
+    @property
+    @memoized
+    def awc_data(self):
+        case_objects = self.row_objects + self.extra_row_objects
+        cases_by_owner = {}
+        for case_object in case_objects:
+            owner_id = case_object.owner_id
+            cases_by_owner[owner_id] = cases_by_owner.get(owner_id, []) + [case_object]
+        return cases_by_owner
+
+    @property
+    def rows(self):
+        rows = []
+        for user in self.users_matching_filter:
+            case_sql_data = self.awc_data.get(user['doc_id'], None)
+            form_sql_data = OpmFormSqlData(DOMAIN, user['doc_id'], self.datespan)
+            row = self.model(user, self, case_sql_data, form_sql_data.data)
+            data = []
+            for t in self.model.method_map:
+                data.append(getattr(row, t[0]))
+            rows.append(data)
+        return rows
+
     def get_row_data(self, row, **kwargs):
-        case_sql_data = OpmCaseSqlData(DOMAIN, row['doc_id'], self.datespan)
```

Comment:
This is the only place `OpmCaseSqlData` is used, so once this is deployed, we should go back and delete it.